CONTRIBUTING.md

```diff
@@ -208,6 +237,12 @@ You can find the coverage report at `./coverage/index.html` after running the te
 
 You can find documentation to create VSCode IDE settings for CodeCatalyst blueprints at [docs/vscode-config.md](./docs/vscode-config.md).
 
+### Find things to do
+
+If you're looking for ideas about where to contribute, consider
+[_good first issue_](https://github.com/aws/aws-toolkit-vscode/labels/good%20first%20issue)
+issues.
```

Comment:
This section was for people brand-new to the repo. If it's not the first item on the page we can just delete it.